src/Discord/Parts/Embed/Embed.php

```diff
@@ -469,6 +504,26 @@ public function setURL(string $url): self
         return $this;
     }
 
+    /**
+     * Checks to see if an url is valid to be used in embeds.
+     *
+     * @param ?string $url
+     *
+     * @return bool
+     */
+    protected function checkEmbedUrl(?string $url, array $allowed = ['http', 'https', 'attachment']): bool
+    {
+        if (null === $url) return true;
+
+        if ($url === '') return false;
```

Comment:
An early return like this is unnecessary. If you attempt to parse the url scheme from an empty string it'd also return `false`